modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -109,6 +126,16 @@ OPTIONS "{" option: value[, ...] "}"
 | Create a vector index on nodes.
 The options map is mandatory because setting the vector dimensions and similarity function is mandatory when creating a vector index.
 
+| [source, syntax, role=noplay, indent=0]
+----
+CREATE VECTOR INDEX [index_name] [IF NOT EXISTS]
+FOR ()-”[“r:TYPE_NAME”]”-()
+ON (r.propertyName)
+OPTIONS "{" option: value[, ...] "}"
+----
+| Create a vector index on relationships.  label:new[Introduced in 5.18]
+The options map is mandatory because setting the vector dimensions and similarity function is mandatory when creating a vector index.
```

Comment:
```suggestion
| Create a vector index on relationships. 
The options map is mandatory because setting the vector dimensions and similarity function is mandatory when creating a vector index. label:new[Introduced in 5.18]
```